Base/usr/share/man/man6/GameOfLife.md

```diff
@@ -14,6 +14,8 @@ $ GameOfLife
 
 GameOfLife is an implementation of John Conway's Game of Life.
 
+Wikipedia: https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life
```

Comment:
It might be neater this way:
```md
GameOfLife is an implementation of [John Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life).
```